std/root.ln

```diff
@@ -173,7 +173,7 @@ export fn toString(n: Result<Stringifiable>): string {
 }
 export fn getOrExit(result: Result<any>): any {
   if result.isErr() {
-    stderrp(result.toString());
+    stderrp(result.getErr(noerr()).toString());
```

Comment:
This works around the fact that I can't `toString` directly all `Result` types, but I can the `Error` type.